packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -452,27 +452,29 @@ describe('Validating rule docs', () => {
               if (token.meta?.includes('skipValidation')) {
                 assert.ok(
                   messages.length === 0,
-                  'Expected not to contain lint errors (with skipValidation):\n' +
-                    token.value,
+                  `Expected not to contain lint errors (with skipValidation):\n${
+                    token.value
+                  }`,
```

Comment:
FWIW I would prefer to turn `\n` into an actual line break since it results in one fewer line, but that might just be my preference.